js/main.js

```diff
@@ -1,16 +1,81 @@
 import {generateOffer} from './util.js';
 import {COUNT_OF_AD} from './data.js';
-import {generateAd, mapBlock} from './popup.js';
-import {deactivatePage, activateForm, activateFilters} from './form.js';
+import {generateAd} from './popup.js';
+import {deactivatePage, activateFilters, activateForm, formBlock} from './form.js';
 
 const keksoAds = new Array(COUNT_OF_AD).fill(null).map(() => generateOffer());
 
-const offerBlock = generateAd(keksoAds[0]);
+deactivatePage();
 
-mapBlock.appendChild(offerBlock);
+//Добавляем карту на страницу
+const cityMap = L.map('map-canvas');
 
-deactivatePage();
+cityMap.on('load', () => {
+  activateForm();
+  activateFilters();
+}).setView([35.680, 139.769], 13);
+
+L.tileLayer(
+  'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
+  {
+    attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors | Icons made by <a href="https://www.freepik.com" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a>',
+  },
+).addTo(cityMap);
+
+//Задаём иконки для нового объявления и похожих, добавляем метки на карту
+const mainIcon = L.icon({
+  iconUrl: '../img/main-pin.svg',
+  iconSize: [52, 52],
+  iconAnchor: [26, 52],
+});
+
+const icon = L.icon({
+  iconUrl: '../img/pin.svg',
```

Comment:
```suggestion
  iconUrl: 'img/pin.svg',
```
А с такими путями картинки нормально отображаются?

Reply:
Да, все работает.

Reply:
Отлично, так они в опубликованной версии тоже работают.